packages/flame/src/Alert/AlertIcons.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { IconWarning } from '../Icon/Warning';
+import { IconInfo } from '../Icon/Info';
+import { IconDanger } from '../Icon/Danger';
+import { IconVerified } from '../Icon/Verified';
+
+interface AlertIcons {
+  type: 'info' | 'success' | 'warning' | 'danger' | string;
+}
```

Comment:
Should we re-use types from https://github.com/lightspeed/flame/pull/82/files#diff-5d1ea7330cf2db21dfc5bfba9fff2b8fR11 here?